app/assets/stylesheets/modules/_container.scss

```diff
@@ -0,0 +1,61 @@
+.drop-container{
+  position: absolute;
+  left: 50%;
+  margin-left: -570px;
+  top: 50%;
+  margin-top: -380px;
+  width: 1140px;
+}
+
+@media (max-height: 870px) {
+ .drop-container{
+    position: relative;
+    left: 0;
+    margin-left: 0;
+    top: 0;
+    margin-top: 0;
+  }
+}
+
+@media (max-width: 1140px) {
+ .drop-container{
```

Comment:
it seems like you have two `.drop-container` classes here... (also one on line 11). Is that on purpose? will one over-ride the other? (I'm asking b/c I'm not a css pro & don't really understand/know what's meant to happen here. I'm learning too!)